### DISCUSSION TITLE
Provide the Manual?

### DISCUSSION THREAD

**Discussion by martinlindenlauf** [mike-edel/ID-MultiPageImporter#32] [General]
Hello, I found en excellent PDF-Manual written by the original Author, Scott Zanelli, coming with a download of an older version (2.5) [provided on creativepro.com](https://creativepro.com/downloads/MultiPageImporter2_5.zip). The manual is written for version 2.5, but as far as I can see the content is still valuable - at least it helped me a lot. I suggest to provide this manual here on GitHub for download. 
As the manual is marked as "This manual is part of the Multi Page Importer bundle" and "Multi Page Importer is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License", providing the manual here should be okay. Because I'm new to Git an topics like that I wanted to ask for feedback before uploading - I could upload it on the Code tab via Add file > Upload, right? @mike-edel could you please check this and give me a short reply. Thank you! Martin

**Comment by mike-edel** [comment]
Thanks! I actually can't remember ever seeing this manual. Let me see how to best include it. There will be a few features that were added after this was written but it should cover 95% of the tool.

**Comment by mike-edel** [comment]
@martinlindenlauf just added it to the code section: https://github.com/mike-edel/ID-MultiPageImporter/blob/master/MultiPageImporterManual.pdf

**Reply by martinlindenlauf** [reply]
Cool, thank you @mike-edel!
